### PR TITLE
[PL - TVP] Fix or add few PL TVP live channels

### PR DESCRIPTION
**Pull request by thomas-ernest**
No PL TVP LIVE channel was working anymore following updates of TVP website.
Some channels could be fixed :
- TVP 3 (every regions)
- TVP World
Some deleted
- TVP Info
- TVP Polonia
Some added
- TVP Alfa
- UA 1
Some might be added, but are also not working :
- TVP 1
- TVP 2

### CODE REVIEW

**Comment by thomas-ernest** [comment]
(!) WARNING : New logos are used for few new channels. It is necessary to merge https://github.com/Catch-up-TV-and-More/resource.images.catchuptvandmore/pull/40 deliver a new version of resource.images.catchuptvandmore and update addon.xml with this new version, BEFORE merging this PR. Review can be started already though.

**Comment by joaopa00** [comment]
Dead code must be removed, not commented. out Please, fix that.

File: addon.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <import addon="script.module.tzlocal" version="2.0.0"/>
     <import addon="script.module.future" version="0.17.1"/>
     <import addon="script.module.kodi-six" version="0.0.4"/>
-    <import addon="resource.images.catchuptvandmore" version="1.0.0"/>
+    <import addon="resource.images.catchuptvandmore" version="1.0.23"/>
```
Comment (thomas-ernest) [inline addon.xml:15]: Planned version based on ongoing activities. 1.0.23 is the one expected after the merge of https://github.com/xbmc/repo-resources/pull/414

File: resources/lib/channels/pl/tvp.py [hunk 2]
```diff
@@ -5,23 +5,22 @@
 # This file is part of Catch-up TV & More
 
 from __future__ import unicode_literals
-import re
-
-from codequick import Resolver, Script, utils
+import json
```
Comment (thomas-ernest) [inline resources/lib/channels/pl/tvp.py:8]: Not sure if it should be defined as a dependency in addon.xml or not ? anyway it seems to be a defaultly or transitively available depenency - missing declaration is not causing bug.

**Comment by thomas-ernest** [comment]
> (!) WARNING : New logos are used for few new channels. It is necessary to merge [Catch-up-TV-and-More/resource.images.catchuptvandmore#40](https://github.com/Catch-up-TV-and-More/resource.images.catchuptvandmore/pull/40) deliver a new version of resource.images.catchuptvandmore and update addon.xml with this new version, BEFORE merging this PR. Review can be started already though.

Anticipated/expected version included in PR. Expected version will be 1.0.23

**Comment by thomas-ernest** [comment]
> Dead code must be removed, not commented. out Please, fix that.

Done. Candidate and deprecated channel removed from PR.

**Comment by joaopa00** [comment]
Some flake8 issue to fix. And do not change the addon.xml file please. It can take months before official kodi repo merge  new version of resources. That will break CUTVM for new comers. If logo is not found, nothing is displayed and CUTVM keeps working .

File: addon.xml (hunk 1 above)
Comment (thomas-ernest) [reply addon.xml]: Restored to 1.0.0 based on Joao's review. See other comments in this PR.

**Comment by thomas-ernest** [comment]
Hello Joao,
New commit added to 

- restore addon.xml
- fix Flake8 linter violations (I was using Pylint, sorry)
- refactor a bit to extract methods and document them.

**Comment by thomas-ernest** [comment]
> It can take months before official kodi repo merge new version

There is a GitHub CI component provided by Kodi to automatize delivery of new version in official repo. It doesn't aceelerate PR review and merge, but it is less effort to streamline new version. When I tag a commit in my repo, it creates a GitHub release and creates PR in official repo.
https://github.com/thomas-ernest/plugin.video.arteplussept/blob/master/.github/workflows/kodi-addon-submitter.yml

**Pull request merged**